pkg/collector/python/init.go

```diff
@@ -252,6 +252,18 @@ func Initialize(paths ...string) error {
 		C.add_python_path(six, C.CString(p))
 	}
 
+	// Setup crash handling specifics - *NIX-only
+	if runtime.GOOS != "windows" {
+		if config.Datadog.GetBool("c_stacktrace_collection") {
+			var cCoreDump int
+
+			if config.Datadog.GetBool("c_core_dump") {
+				cCoreDump = 1
+			}
+			C.handle_crashes(six, C.int(cCoreDump))
```

Comment:
cgo fails to find this symbol because it's behind a preprocessor directive. 
Instead of detecting the OS at runtime we should probably move this bit behind a Go build flag (or `foo_win.go` module).

Reply:
Absolutely right 🤦‍♂ Thanks for the feedback!